src/bundle/Resources/public/scss/_typography.scss

```diff
@@ -0,0 +1,37 @@
+h1 {
+    font-size: 1.75rem;
+}
+
+h2 {
+    font-size: 1.5rem;
+}
+
+h3 {
+    font-size: 1.375rem;
+}
+
+h4 {
+    font-size: 1.25rem;
+}
+
+h5 {
+    font-size: 1.125rem;
```

Comment:
This `hx` font-size can be set by bootstrap variables. In `_custom.scss` you should set variables, for default in boostrap it is: 
```
$h1-font-size: 2.5rem !default;
$h2-font-size: 2rem !default;
$h3-font-size: 1.75rem !default;
$h4-font-size: 1.5rem !default;
$h5-font-size: 1.25rem !default;
$h6-font-size: 1rem !default;
```